README.md

```diff
@@ -2539,6 +2539,37 @@ no parameters.
   end
   ```
 
+* <a name="accessor_mutator_method_names"></a>
+  For more complex accessors and mutators, avoid prefixing methods with
+  `get_` and `set_`.
+  It is a ruby convention to use direct names for accessors (readers) and
```

Comment:
However its worth mentioning that overriding attribute accessors in Rails
can easily backfire at you. In sich cases I prefer to use get_/set_.
5 cze 2016 09:08 "Bozhidar Batsov" <notifications@github.com> napisał(a):

> In README.md
> <https://github.com/bbatsov/ruby-style-guide/pull/582#discussion_r65812358>
> :
>
> > @@ -2539,6 +2539,37 @@ no parameters.
> >    end
> >    ```
> >
> > +* <a name="accessor_mutator_method_names"></a>
> > +  For more complex accessors and mutators, avoid prefixing methods with
> > +  `get_` and `set_`.
> > +  It is a ruby convention to use direct names for accessors (readers) and
>
> direct names -> attribute names (or something like this)
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/bbatsov/ruby-style-guide/pull/582/files/a9c6b9a14a33fc8671c7080350e0afdde52da42c#r65812358>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AJW3u7DicBuflwlE307qS6_E4a_WeKmGks5qInXwgaJpZM4ItQ5J>
> .
>